docs/whatsnew-5.1.rst

```diff
@@ -290,6 +290,10 @@ you should import `kombu.utils.encoding` instead.
 If you were using the `celery.task` module before, you should import directly
 from the `celery` module instead.
 
+If you were using the `celery.task` decorator you should use 
+`celery.shared_task` instead.
```

Comment:
or Task instead

Reply:
Task is not a decorator, this does not seem to work?

Reply:
I can add another line for completenes sake: 

```
If you were using `from celery.task import Task` you should use `from celery import Task` instead.
```

Reply:
you are right